src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -279,13 +279,12 @@ public void Rotate(Dictionary<string, int> opts)
 
         public void BackgroundApp() =>
             Execute(AppiumDriverCommand.BackgroundApp,
-                AppiumCommandExecutionHelper.PrepareArgument("seconds",
-                    AppiumCommandExecutionHelper.PrepareArgument("timeout", null)));
+                AppiumCommandExecutionHelper.PrepareArgument("seconds", null));
```

Comment:
I suppose the default value should be `-1`, so the app is put into background forever.

Reply:
Done